modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -55,10 +55,13 @@ If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a s
 [id="understanding-self-node-remediation-remediation-template-config_{context}"]
 == Understanding the Self Node Remediation Template configuration
 
-The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR with the name `self-node-remediation-default-template` in the Self Node Remediation Operator's namespace. This CR defines the remediation strategy for the nodes.
+The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR. This CR defines the remediation strategy for the nodes.
 
-The default remediation strategy is `NodeDeletion` that removes the `node` object.
-In {product-title} 4.10, the Self Node Remediation Operator introduced a new remediation strategy called `ResourceDeletion`. The `ResourceDeletion` remediation strategy removes the pods and associated volume attachments on the node rather than the `node` object. This strategy helps to recover workloads faster.
+The following remediation strategies are available:
+
+ResourceDeletion:: This strategy remediation strategy removes the pods and associated volume attachments on the node rather than the node object. This strategy helps to recover workloads faster. `ResourceDeletion` is the default remediation strategy. It creates the `self-node-remediation-resource-deletion-template`.
+
+NodeDeletion:: This remediation strategy removes the node object. It creates the `self-node-remediation-node-deletion-template`.
```

Comment:
maybe will be better:

- It uses the `self-node-remediation-node-deletion-template`.

instead of:

- It creates the self-node-remediation-node-deletion-template